### ISSUE TITLE
Fix dimension of localized strings 

### ISSUE THREAD

**Issue by Airon90** [Tuxemon/Tuxemon#242]
See the picture:

![schermata da 2016-08-26 13-30-31](https://cloud.githubusercontent.com/assets/139841/18005409/157c3b90-6b9a-11e6-9895-f0a78684e1fe.png)

The arrow in the 4-option menu is covering part of "Combatti" ("Fight") label.
Is this a translation bug (should the Italian translators (me :D) find something different?) or a design bug (should the interface be changed to adapt it to the translations?)?

**Comment by bitcraft** [comment]
If you can use a shorter word for Combatti, then that would be the easiest fix.  If its not possible, let's leave this open and see if we can figure out a simple way to resize menus when strings are too large.

**Comment by Airon90** [comment]
In Italian "Lotta" is another word that can be used there but [it is already used in Pokémon games](http://www.gamesailors.it/public/news/Pokemon%20Rosso-Blu/Pokemon_Blue_3.jpg). A tricky way is  to read the word and to invent an algorithm that cut the word and put a dot at the end of the word ("Combatti" could become "Combat.")

**Comment by Airon90** [comment]
This bug is useful in order to create some translation guidelines too.

**Comment by bitcraft** [comment]
We have a wiki that is better suited for translation guidelines.  https://github.com/Tuxemon/Tuxemon/wiki/Translations

**Comment by Airon90** [comment]
I know that there is a page in the wiki but there isn't anything like: «keep the translations of keys X, Y, and Z as short as possible» or «key X is an adjective used by key Y» (I refer to #236 :P).
By now that page is just an how-to for creating a new translation and using it. ;)

**Comment by xirsoi** [comment]
Yes, but the a "Style Guide" type tying could live on that page as it's own section.

**Comment by bitcraft** [comment]
The wiki is open to edit if anyone wants to add things like that.  I'd caution not to do too much work documenting it though, as our translation guidelines are inevitably going to change.

**Comment by bitcraft** [comment]
A proper fix to this will be a feature to adjust the containing widget/border to the size of the children, with an anchor.

**Comment by Qiangong2** [comment]
Could we also just reduce the padding in the choice box? That might fix it in the short term

**Comment by bitcraft** [comment]
That would fix it in the short term, but this might be raised again for another language.  I'd rather just leave it open until a more permanent/smart fix is implemented, rather than chasing small fixes.
